account.go

```diff
@@ -0,0 +1,33 @@
+package quickbooks
+
+import (
+	"fmt"
+)
+
+type AccountQueryResponse struct {
+	Accounts []*Account `json:"Account"`
+}
+
+type Account struct {
+	ID      string `json:"Id"`
+	Name    string `json:"Name"`
+	Active  bool   `json:"Active"`
+	Type    string `json:"AccountType"`
+}
+
+func (c *Client) FetchAccounts(sql string) ([]*Account, error) {
```

Comment:
This shouldn't take a SQL string, it should take some filter struct that allows the user to filter by Type and IsActive. In other words, declare something like:

```
type AccountFIlter struct {
	Type string
	IsActive bool
}
```